infra/build/developer-tools-light/Dockerfile

```diff
@@ -29,6 +29,9 @@ RUN apt-get update -y && apt-get install -y -q \
 # Install Terraform
 COPY --from=installer /software/terraform /usr/local/bin/terraform
 
+# bash is preferred over /bin/sh
+RUN apk add --no-cache bash parallel
```

Comment:
Why do we need a separate step, we do an apk add above and these can be appended there?

Reply:
Fixed